src/services/completions.ts

```diff
@@ -3482,6 +3483,9 @@ function getCompletionData(
             }
             addTypeProperties(type, !!(node.flags & NodeFlags.AwaitContext), insertQuestionDot);
         }
+        else if (isInTypeQuery(node) && !type.isNullableType()) {

```

Comment:
hm, perhaps this shouldn't even check if the type is not nullable since this autocomplete just fine (it just reports an error afterward):
```ts
declare const obj: { a: { b: string } } | undefined

type Test = typeof obj./**/
```